pilot/pkg/bootstrap/server.go

```diff
@@ -550,6 +562,24 @@ func (s *Server) createK8sServiceControllers(serviceControllers *aggregate.Contr
 				})
 		}
 	}
+
+	return
+}
+
+// initMultiClusterController initializes multi cluster controller
```

Comment:
Do we still need the old logic in line 542? I think it can be handled by dynamic logic here.